examples/all-clusters-app/esp32/main/LEDWidget.cpp

```diff
@@ -83,6 +83,10 @@ void LEDWidget::Init(gpio_num_t gpioNum)
             .timer_sel  = LEDC_TIMER_1,
             .duty       = 0,
             .hpoint     = 0,
+            {
+                .output_invert = 1,
```

Comment:
With this change, does the code work fine for ESP32-C3 too (that has an onboard LED)